specs/wp-invite-users-spec.js

```diff
@@ -286,6 +271,128 @@ testDescribe( `[${ host }] Invites:  (${ screenSize })`, function() {
 		} );
 	} );
 
+	test.describe.only( 'Inviting New User as an Editor And Revoke Invite: @parallel @jetpack', function() {
```

Comment:
@sofiavistas Does `.only` should stay here? I do not know will CircleCI skip the other tests from this spec. Maybe somebody can confirm this?

Reply:
Good catch @Stojdza, yes, CircleCI will skip the other tests, so this should be removed